styles.css

```diff
@@ -0,0 +1,53 @@
+.header{
+  display: flex;
+  background-color: rgb(161, 167, 167);
+  height: auto;
+}
+h1{
+  margin-left: 20px;
+}
+.search{
+ 
+  display: flex;
+  margin-top: 20px;
+  margin-left: 250px;
+}
+.search-input{
+  display: block;
+  height: 35px;
+  width: 500px;
+  
+}
```

Comment:
The best practice and standard is to always have a space before and after the curly braces of the CSS selectors